tests/integration/components/model-picker-test.js

```diff
@@ -180,19 +180,20 @@ module('Integration | Component | {{model-picker}}', function(hooks) {
       assert.equal(query.filter.code_cont, 'test', 'queries with filter');
     };
 
-    this.owner.register('service:store', Service.extend({
+    this.set('store', {
```

Comment:
Why change this? The component should work with the service, not with a passed argument

Reply:
I tried to fix this, but when I change back it keeps failing and I couldn't figure out why.